Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealitySpatialAwarenessSystemProfile.asset

```diff
@@ -12,7 +12,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 19f279aded72cb741b4de89a54359dd4, type: 3}
   m_Name: DefaultMixedRealitySpatialAwarenessSystemProfile
   m_EditorClassIdentifier: 
-  isCustomProfile: 1
+  isCustomProfile: 0
```

Comment:
Note to self - I can update the presubmit script to ensure this doesn't happen again (i.e. check that none of the checked in profiles has isCustomProfile: 1